packages/carbon-react/src/components/Pagination/Pagination.stories.js

```diff
@@ -0,0 +1,88 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import { action } from '@storybook/addon-actions';
+
+import {
+  withKnobs,
+  array,
+  boolean,
+  number,
+  text,
+} from '@storybook/addon-knobs';
+import { Pagination } from 'carbon-components-react';
+
+const props = () => ({
+  disabled: boolean('Disable page inputs (disabled)', false),
+  page: number('The current page (page)', 1),
+  totalItems: number('Total number of items (totalItems)', 103),
+  pagesUnknown: boolean('Total number of items unknown (pagesUnknown)', false),
+  pageInputDisabled: boolean(
+    'Disable page input (pageInputDisabled)',
+    undefined
+  ),
+  pageSizeInputDisabled: boolean(
+    'Disable page size input (pageSizeInputDisabled)',
+    undefined
+  ),
+  backwardText: text(
+    'The description for the backward icon (backwardText)',
+    'Previous page'
+  ),
+  forwardText: text(
+    'The description for the forward icon (forwardText)',
+    'Next page'
+  ),
+  pageSize: number('Number of items per page (pageSize)', 10),
+  pageSizes: array('Choices of `pageSize` (pageSizes)', [10, 20, 30, 40, 50]),
+  itemsPerPageText: text(
+    'Label for `pageSizes` select UI (itemsPerPageText)',
+    'Items per page:'
+  ),
+  onChange: action('onChange'),
+});
```

Comment:
Same as above. Can we just add the necessary ones to the `Pagination` component directly?